src/Kazoo/Common/Utils.php

```diff
@@ -18,6 +18,9 @@ public static function shortClassName($class) {
      */
     public static function underscoreClassName($class) {
         $class_name = self::shortClassName($class);
+        if ($class_name == "Phonenumber") { 
```

Comment:
Just name the classes PhoneNumber and PhoneNumbers, then you don't need to do this.